modules/cxf/src/main/java/org/mule/module/cxf/config/CxfNamespaceHandler.java

```diff
@@ -48,6 +49,17 @@ protected String resolveId(Element element,
                 return CxfConstants.DEFAULT_CXF_CONFIGURATION;
             }
 
+            @Override
+            protected AbstractBeanDefinition parseInternal(Element element, ParserContext parserContext)
+            {
+                // We don't want spring managing lifecycle anymore, but since this module
+                // is going to be rewritten, it doesn't make sense to spend time in removing this
+                // init method
+                AbstractBeanDefinition beanDefinition = super.parseInternal(element, parserContext);
+                beanDefinition.setInitMethodName(Initialisable.PHASE_NAME);
```

Comment:
Seems like this change can be required in users's custom code too. Please document if true